### DISCUSSION TITLE
Separate Discord channel option for Mod Updates (or Updates in general)

### DISCUSSION THREAD

**Discussion by mixels** [gameserverapp/Platform#977] [Suggestions]
Hi! In Discord Bot configuration, I would please like to request the option to configure a channel specifically for mod updates or possibly game + mod updates combined.

In Steam-based games, knowing when a server is updated is very helpful to players because Steam Workshop doesn't reliably auto update mods. Players often need to manually unsubscribe and resubscribe to mods to force the mods to update. Knowing what mods have updated on a server helps the players know they need to unsubscribe and resubscribe.

With game updates, the use case is more about players knowing that server has updated. When we write game server logs to a single channel, often the game update notifications can get buried under countdown, server halted, and server online notifications. Having a more focused channel that shows only server updates would declutter and help players find that info that's most useful to them more easily.

**Comment by mvaessen** [comment]
Thanks for the suggestion, this is now available.
